ivy_tests/test_ivy/test_frontends/test_torch/test_reduction_ops.py

```diff
@@ -330,6 +330,35 @@ def test_torch_mean(
     )
 
 
+# median
+@handle_cmd_line_args
+@given(
+    dtype_and_x=helpers.dtype_and_values(
```

Comment:
You will need to generate odd-sized arrays as it is indicated in [documentation](https://pytorch.org/docs/stable/generated/torch.median.html) to have the tests pass as the behavior of `torch.median` doesn't basically compute `quantile` with `q=0.5` for even sized arrays.

Reply:
Hello! Thank you for your feedback. I don't understand how can I generate odd-sized arrays. Can you please guide me to any resource? Is it going to be better if I handled the even-sized arrays inside the median function?

Reply:
Hey well yeah, we have example tutorials on fixing unit tests that can be found on Ivy's youtube channel and can help you to understand more about testing, your idea of handling even-sized arrays can be a good option and it can allow this frontend function to behave like the original PyTorch function. Feel free to write a composition for it. Referring to the frontend guide here [Frontend Guide](https://lets-unify.ai/ivy/deep_dive/16_ivy_frontends.html#compositions)
Thanks again🙂

Reply:
In order for this function to behave like the original Pytorch function, I will need to use ivy.quantile() which is not implemented yet, it is still here https://github.com/unifyai/ivy/issues/3856. Also, if I tested the function with only odd-sized arrays, it will pass the test but it will only be a temporary solution and it will not solve the problem, since if someone is using the ivy library and called it with an even-sized array, it will not behave as expected. Please correct me if I am wrong. 

Should I leave this function and try to implement something else?

Reply:
Yea you are right if you implement it right now it will be a temporary implementation 🙂, yet welcome for the contribution😊
If it is a complicated function you can leave it and go on to work on the task you are comfortable with.
Thanks🙂